packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,15 +104,28 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
       decoration: BoxDecoration(
         border: Border.all(color: Theme.of(context).focusColor),
       ),
-      child: InspectorTree(
-        controller: summaryTreeController,
-        isSummaryTree: true,
+      child: GestureDetector(
+        child: InspectorTree(
+          controller: summaryTreeController,
+          isSummaryTree: true,
+        ),
+        onTapUp: (TapUpDetails tapUpDetails) {
```

Comment:
This works as long as I click in the blank space of the tree. If I click on an actual node it doesn't fire.

Reply:
try modifying the `behavior` property of the GestureDetector. Maybe `HitTestBehavior.opaque`?

Reply:
That didn't seem to have any effect. I added a line in the node's onTap() to explicitly focus the tree, along with a TODO to looks for a better way.